### DISCUSSION TITLE
I need code that runs through all the frames and performs my action

### DISCUSSION THREAD

**Discussion by FreeM1ne** [kaliiiiiiiiii/Selenium-Driverless#197] [Q&A] [answered]
I need code that runs through all the frames and performs my action

**Comment by ttlns** [comment, marked as answer]
Yeah well I mean - feel free to develop that then?

**Reply by FreeM1ne** [reply]
I think you should shut down the project. You have no interest.
A lot of problems that you're not even trying to fix.
